Keda.Cosmosdb.Scaler/src/Services/CosmosDbTrigger.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Keda.CosmosDB.Scaler.Services
+{
+    public class CosmosDBTrigger
+    {
+        public string CosmosDBConnectionString { get; internal set; }
+        public string DatabaseName { get; internal set; }
+        public string CollectionName { get; internal set; }
+        public string AccountName { get; internal set; }
+        public CosmosDBLease Lease { get; internal set; }
+
+        public CosmosDBTrigger()
+        { 
+        }
+
+        public CosmosDBTrigger(string connectionString, string databaseName, string collectionName, string accountName)
+        {
+            CosmosDBConnectionString = connectionString;
+            DatabaseName = databaseName;
+            CollectionName = collectionName;

```

Comment:
Add validation similar to https://github.com/Azure/azure-webjobs-sdk-extensions/blob/ac8226ba786fe041cf215889d83880961bef93b0/src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttribute.cs#L23-L34

Add unit tests - that verify exceptions as well

Reply:
We already have that when we read the scaler metadata. And we have unit tests for that as well.